validation-test/stdlib/FixedPointConversion.swift.gyb

```diff
@@ -97,6 +124,105 @@ FixedPointConversionFailure.test("${Other}To${Self}Conversion/dest=${testValue}"
 %     end # for testValue in ...
 %   end # for in all_integer_types (Other)
 % end # for in all_integer_types (Self)
+
+%   for Other, otherSignificandBits in floatNameToSignificandBits.iteritems():
```

Comment:
Did you mean it to be nested inside `for self_ty in all_integer_types(word_bits):`?